wrappers/javascript/indy-vdr-nodejs/src/NodeJSIndyVdr.ts

```diff
@@ -141,6 +141,11 @@ export class NodeJSIndyVdr implements IndyVdr {
     this.handleError(this.nativeIndyVdr.indy_vdr_set_cache_directory(path))
   }
 
+  public setLedgerTxnCache(options: { capacity: number; expiry_offset_ms: number; path: string }): void {
```

Comment:
the path argument here could be optional `path?: string` but I'm not sure how well that would cross the Typescript => C => Rust function interfaces. Thought? @andrewwhitehead

Reply:
undefined will be converted into a nullptr. Should be fine to keep the type at `path?: string`.